test/property.cpp

```diff
@@ -121,135 +123,55 @@ BOOST_AUTO_TEST_CASE( shared_subscription_available ) {
 
 // property has _ref
 
-BOOST_AUTO_TEST_CASE( content_type_ref ) {
-    mqtt::v5::property::content_type v1 { "abc" };
-    mqtt::v5::property::content_type_ref v_ref1 { "abc" };
-
-    mqtt::v5::property::content_type v2 { v_ref1 };
-    mqtt::v5::property::content_type_ref v_ref2 { v1 };
-
-    BOOST_TEST(v2.val() == v_ref2.val());
-
+BOOST_AUTO_TEST_CASE( content_type ) {
+    mqtt::v5::property::content_type v1 { "abc"_mb };
     BOOST_TEST(boost::lexical_cast<std::string>(v1) == "abc");
```

Comment:
I don't understand how boost::lexical_cast knows what to do with mqtt::v5::property::content_type ...?

Can you help me understand?

Reply:
This is a test for 

```cpp
std::ostream& operator<<(std::ostream&, mqtt::v4::property::content_type const&);
```

lexical_cast uses stream output operator. https://www.boost.org/doc/libs/1_71_0/doc/html/boost_lexical_cast/synopsis.html#boost_lexical_cast.synopsis.lexical_cast

It is easy to write test codes so I use lexical_cast.

Reply:
Ah, I did not know that. Thank you for explaining.